### PR TITLE
Bug 1014915 - Fix example server

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: example/server.js [hunk 1]
```diff
@@ -1,39 +1,65 @@
 var express = require('express');
-var Habitat = require('habitat');
+var i18n = require("webmaker-i18n");
```
Comment (k88hudson) [inline example/server.js]: quotes

File: example/views/index.nunjucks [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+  <meta charset="utf-8">
+  <title>Persona example</title>
+  <link href="https://s3-us-west-2.amazonaws.com/makerstrap/makerstrap.complete.min.css" rel="stylesheet">
+  <link href="../create-user/create-user-form.css" rel="stylesheet">
```
Comment (k88hudson) [inline example/views/index.nunjucks]: You really shouldn't include both the create user form css and makerstrap, the latter is a shim of the former

Comment (k88hudson) [inline example/views/index.nunjucks]: So for this example, remove makerstrap

File: example/server.js [hunk 3]
```diff
@@ -1,39 +1,65 @@
...
+    res.render('index.nunjucks');
+  });
+
+  app.post('/verify', login.handlers.verify);
+  app.post('/authenticate', login.handlers.authenticate);
+  app.post('/create', login.handlers.create);
+  app.post('/logout', login.handlers.logout);
+  app.post('/check-username', login.handlers.exists);
+
+  return app;
```
Comment (k88hudson) [inline example/server.js:64]: Can you explain in the readme or in comments that this example applies to Webmaker apps that use i18n and nunjucks?

File: example/views/index.nunjucks (hunk 2 above)
Comment (jbuck) [inline example/views/index.nunjucks]: If I remove either of these CSS files, the example doesn't display correctly.

If I remove makerstrap, when the create user modal appears I get a large black background and I can't see anything. If I remove the create-user-form css then I get no styling on the buttons

Comment (k88hudson) [inline example/views/index.nunjucks]: We should fix the create-user-form then

File: example/server.js (hunk 3 above)
Comment (jbuck) [inline example/server.js:64]: Done! I think

**Pull request merged**